Buddies/Utilities/DataAccessor.swift

```diff
@@ -217,24 +219,25 @@ class DataAccessor : LoggedInUserInvalidationDelegate, ActivityInvalidationDeleg
         }
 
         _userRegistration[id] = usersCollection.document(id).addSnapshotListener {
-            guard let snap = $0 else {
-                print($1!)
-                return
-            }
-            
-            guard let user = OtherUser.from(snap: snap) else {
-                print("invalid user :( \(id)")
-                return
-            }
+            var user: OtherUser?
             
-            self.storageManager.getImage(imageUrl: user.imageUrl, localFileName: user.uid) { img in
-                user.image = img
-                self.onInvalidateUser(user: user)
+            if let snap = $0 {
```

Comment:
Even though you're assigning a name to `$0` here, I would recommend considering named parameters.  Outside of short closures, I think `$0` style arguments can be confusing.

Reply:
good call